packages/lib/shim-init-node.ts

```diff
@@ -309,13 +310,11 @@ function shimInit(options: ShimInitOptions = null) {
 
 		const isUpdate = !!options.destinationResourceId;
 
-		const uuid = require('./uuid').default;
-
 		if (!(await fs.pathExists(filePath))) throw new Error(_('Cannot access %s', filePath));
 
 		defaultProps = defaultProps ? defaultProps : {};
 
-		let resourceId = defaultProps.id ? defaultProps.id : uuid.create();
+		let resourceId = defaultProps.id ? defaultProps.id : BaseItem.generateUuid();
```

Comment:
This change is also necessary to make snapshot works, and it working because of the change added to `BaseModel`